tests/test_index.py

```diff
@@ -649,11 +687,15 @@ def _build_test_index(workdir):
 
     assert "revoke_test-1.0-0.tar.bz2" in pkg_list
     assert "zlib" in pkg_list["revoke_test-1.0-0.tar.bz2"]["depends"]
-    assert "package_has_been_revoked" not in pkg_list["revoke_test-1.0-0.tar.bz2"]["depends"]
+    assert (
+        "package_has_been_revoked"
+        not in pkg_list["revoke_test-1.0-0.tar.bz2"]["depends"]
+    )
 
     assert "remove_test-1.0-0.tar.bz2" in pkg_list
 
 
+# SLOW
```

Comment:
How about we add a custom slow marker?

https://docs.pytest.org/en/6.2.x/mark.html

Let's use pytest's full capabilities 💪